### PR TITLE
replace _stricmp with storm::iEquals

### PR DESCRIPTION
**Pull request by q4a**
same for _strcmpi

### CODE REVIEW

File: src/libs/common/include/attributes.h [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #include <vector>
 
 #include "storm_assert.h"
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/common/include/attributes.h]: should be "storm/string_compare.hpp"
`util` must be required in cmakefile

File: src/libs/animation/src/action_info.cpp [hunk 2]
```diff
@@ -10,6 +10,7 @@
 
 #include "action_info.h"
 #include "storm_assert.h"
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/animation/src/action_info.cpp]: same here

File: src/libs/animation/src/action_player_imp.cpp [hunk 3]
```diff
@@ -10,6 +10,8 @@
 
 #include "animation_imp.h"
 
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/animation/src/action_player_imp.cpp]: same here

File: src/libs/animation/src/animation_info.cpp [hunk 4]
```diff
@@ -10,6 +10,8 @@
 
 #include "animation_info.h"
 
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/animation/src/animation_info.cpp]: same

File: src/libs/geometry/src/geom_static.cpp [hunk 5]
```diff
@@ -13,6 +13,8 @@ Import library main file
 #include <cstring>
 #include <vector>
 
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/geometry/src/geom_static.cpp:16]: same

File: src/libs/lighter/src/lighter_lights.cpp [hunk 6]
```diff
@@ -10,6 +10,8 @@
 
 #include "lighter_lights.h"
 
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/lighter/src/lighter_lights.cpp]: same

File: src/libs/location/src/locator_array.cpp [hunk 7]
```diff
@@ -10,6 +10,8 @@
 
 #include "locator_array.h"
 
+#include "../../util/include/storm/string_compare.hpp"
```
Comment (espkk) [inline src/libs/location/src/locator_array.cpp]: same

**Review by espkk — CHANGES_REQUESTED** [review]
Looks nice! But please take a look at include paths. they should not contain "../"
for concerned libraries `util` shall be required in cmakelists.txt like this
```STORM_SETUP(
    TARGET_NAME common
    TYPE library
    DEPENDENCIES shared_headers *util* spdlog directx
)

File: src/libs/common/include/attributes.h (hunk 1 above)
Comment (q4a) [reply src/libs/common/include/attributes.h]: fixed

File: src/libs/animation/src/action_info.cpp (hunk 2 above)
Comment (q4a) [reply src/libs/animation/src/action_info.cpp]: fixed

File: src/libs/animation/src/animation_info.cpp (hunk 4 above)
Comment (q4a) [reply src/libs/animation/src/animation_info.cpp]: fixed

File: src/libs/lighter/src/lighter_lights.cpp (hunk 6 above)
Comment (q4a) [reply src/libs/lighter/src/lighter_lights.cpp]: fixed

File: src/libs/location/src/locator_array.cpp (hunk 7 above)
Comment (q4a) [reply src/libs/location/src/locator_array.cpp]: fixed

File: src/libs/geometry/src/geom_static.cpp (hunk 5 above)
Comment (q4a) [reply src/libs/geometry/src/geom_static.cpp:16]: fixed

File: src/libs/animation/src/action_player_imp.cpp (hunk 3 above)
Comment (q4a) [reply src/libs/animation/src/action_player_imp.cpp]: fixed

**Comment by q4a** [comment]
> Looks nice! But please take a look at include paths. they should not contain "../" for concerned libraries `util` shall be required in cmakelists.txt

Fixed. Thanks.

**Review by espkk — APPROVED** [review]

**Pull request merged**